vision_agent/tools/tools.py

```diff
@@ -1665,6 +1665,80 @@ def closest_box_distance(
     return cast(float, np.sqrt(horizontal_distance**2 + vertical_distance**2))
 
 
+def flux_image_inpainting(
+    prompt: str,
+    image: np.ndarray,
+    mask_image: np.ndarray,
+    height: Optional[int] = 512,
+    width: Optional[int] = 512,
+    strength: Optional[float] = 0.6,
+    num_inference_steps: Optional[int] = 10,
+    guidance_scale: Optional[float] = 3.5,
+    seed: Optional[int] = None,
+) -> List[np.ndarray]:
+    """'flux_image_inpainting' is a tool that uses a rectified flow transformer capable of
+    inpainting images. It receives a prompt, an image, a mask and returns the inpainted image.
+    The mask white pixels in the mask are repainted while black pixels are preserved.
+
+    Parameters:
+        prompt (str): The question about the image
+        image (np.ndarray): The reference image used for the question
+        mask_image (np.ndarray): The image representing the mask.
+        height (Optional[int]): The height of the generated image. Defaults to 512.
+        width (Optional[int]): The width of the generated image. Defaults to 512.
+        strength (Optional[float]): The strength of the inpainting. Defaults to 0.6.
+        num_inference_steps (Optional[int]): The number of inference steps. Defaults to 10.
+        guidance_scale (Optional[float]): The guidance scale. Defaults to 3.5.
+        seed (Optional[int]): The seed for the random number generator. Defaults to None generating a random seed.
```

Comment:
given that this goes into the prompt eventually, I would advise to make the text more descriptive. for example, what does strength mean and what are the allowed values? It should say something like: "when strength is 1.0 it ignores the provided image..." or something like that. and same for the rest of ambiguous parameters.

Reply:
Makes total sense! Improved the docs